apps/wallet-mobile/src/TxHistory/TxHistoryNavigator.tsx

```diff
@@ -136,34 +131,23 @@ export const TxHistoryNavigator = () => {
                   options={{
                     title: walletName ?? '',
                     headerRight: headerRightHistory,
-                    headerStyle: {
-                      elevation: 0,
-                      shadowOpacity: 0,
-                      backgroundColor: '#E1EAF6',
-                    },
                   }}
                 />
 
-                <Stack.Screen name="history-details" options={{title: ''}}>
+                <Stack.Screen name="history-details" options={{ title: '' }}>
                   {() => (
-                    <Boundary loading={{size: 'full'}}>
+                    <Boundary loading={{ size: 'full' }}>
                       <TxDetails />
                     </Boundary>
                   )}
                 </Stack.Screen>
 
                 <Stack.Screen
                   name="receive"
-                  component={ReceiveScreen}
+                  component={ReceiveScreenNavigator}
```

Comment:
adding an extra navigator will require to manage the navbar if the children nav is also a stack, and will also create some complications to customize the navbar if the presented one is the grand-parent and not the parent, maybe keep the routes within the history for now since it's launched from here?

Reply:
cc: @SorinC6 @euvini